test/message/promise_fast_handled_reject.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+
+const p1 = new Promise(function(res, rej) {
+  consol.log('One'); // eslint-disable-line no-undef
+});
+
+new Promise(function(res, rej) {
```

Comment:
So split it into two lines or use an arrow function :P? 

I think this is a critical part in the test, no?